src/examples.md

```diff
@@ -918,13 +918,13 @@ We recommend viewing Example 1 before viewing this example._
 
 
 <div id="ex27" markdown="1">
-##Example 27: Laplace Boundary Conditions
+##Example 27: Boundary Conditions
```

Comment:
This is not very descriptive. I would revert this as it is a special kind of BC.

Reply:
I think he meant it reads like it's a special type of BC.

Either of the two elaborate titles you proposed should work.

Reply:
I like "Boundary Conditions for Elliptic Problems"

Reply:
Done!

Reply:
I made the change while the last comment was being written. I am also happy to use "boundary conditions for the Laplace problem"